projects/api-client/src/model/personDto.ts

```diff
@@ -34,9 +35,11 @@ export interface PersonDto {
   firstName: string;
   lastName?: string;
   nickname?: string;
```

Comment:
why did you edit this file? we are not using this (api-client). All the models are in the models folders

Reply:
Just wanted to keep it updated until we have it. It is a bit confusing when you can find 2 files for the same purpose even if it's not used, at least let's have it up to date, or delete it.

Reply:
Yep, it cannot be updated. It will give errors like this at build time. I think we should have a separate ticket to delete these files.